File: src/test/resources/Fig3Test.kt
import kotlin.test.Test
import kotlin.test.assertEquals

class C(val x: Int) {
}

class Fig3Test {

    @Test
    fun testFig3Value0() {
        val actual = C(0).x
        assertEquals(0, actual)
    }

    @Test
    fun testFig3ValueNegative() {
        val actual = C(-1).x
        assertEquals(-1, actual)
    }

    @Test
    fun testFig3ValueMax() {
        val actual = C(2_147_483_647).x
        assertEquals(2_147_483_647, actual)
    }

}

File: swift_results/Fig3Test.swift
import XCTest
#if false
let Ꮻimports = ["kotlin.test.Test", "kotlin.test.assertEquals"]
#endif

class C {
    let x: Int
    init(x: Int) {
        self.x = x
    }
}

class Fig3Test: XCTestCase {
    func testFig3Value0() {
        let actual = C(x: 0).x
        XCTAssertEqual(actual, 0)
    }

    func testFig3ValueNegative() {
        let actual = C(x: -1).x
        XCTAssertEqual(actual, -1)
    }

    func testFig3ValueMax() {
        let actual = C(x: 2_147_483_647).x
        XCTAssertEqual(actual, 2_147_483_647)
    }
}